cmd/talosctl/cmd/talos/list.go

```diff
@@ -148,13 +149,23 @@ var lsCmd = &cobra.Command{
 					}
 				}
 
-				fmt.Fprintf(w, "%s\t%s\t%d\t%d\t%s\t%s\t%s\n",
+				label := ""
+				if info.Xattrs != nil && len(info.Xattrs) > 0 {
```

Comment:
Like for me - no need to add `len(info.Xattrs) > 0`

The check `len(info.Xattrs) > 0` is redundant because of the preceding condition `info.Xattrs != nil` already ensures that `info.Xattrs` is not nil. If `info.Xattrs` is not nil, then it is safe to iterate over it, even if it is an empty slice.

Reply:
So `if info.Xattrs != nil {` will be enough

Reply:
Thank you, fixed!